smoke-tests/designer/features/steps/completeAForm.steps.js

```diff
@@ -35,5 +36,45 @@ Then("I taken directly to the page titled {string}", function (pageTitle) {
 
 Given("I have progressed to the Do you have any evidence? page", function () {
   formRunner.selectRadio("Yes, I do have a link");
-  formRunner.multilineText("https://nodejs.org/en/");
+  formRunner.textBox("https://nodejs.org/en/");
+});
+
+Then(/^the Summary page is displayed with my answers$/, function () {
+  browser.waitUntil(
+    () => formRunner.pageTitle.getText().toLowerCase() === "summary",
+    {
+      timeoutMsg: `Failed to reach the Summary page, page url is: ${browser.getUrl()}`,
+    }
+  );
+  expect(formRunner.summaryAnswer(formData.yesNo.question)).toHaveText("Yes");
+  expect(formRunner.summaryAnswer(formData.address.question)).toHaveText(
+    "Testington Manor, Test Road, Test Town, TT1 1TT"
+  );
+  expect(formRunner.summaryAnswer(formData.dateField.question)).toHaveText(
+    "1 January 2020"
+  );
+  expect(formRunner.summaryAnswer(formData.checkBox1.question)).toHaveText(
+    "1, 2"
+  );
+  expect(formRunner.summaryAnswer(formData.autoComp.question)).toHaveText(
+    "Not supplied"
+  );
+  expect(formRunner.summaryAnswer(formData.textField.question)).toHaveText(
+    "740"
+  );
+  expect(formRunner.summaryAnswer(formData.dateParts.question)).toHaveText(
+    "22 March 2021"
+  );
+  expect(formRunner.summaryAnswer(formData.radio2.question)).toHaveText(
+    "Not supplied"
+  );
+  expect(formRunner.summaryAnswer(formData.multiLine.question)).toHaveText(
+    "I've turned it into a spaceship capable of interstellar travel"
+  );
+  expect(formRunner.summaryAnswer(formData.textField2.question)).toHaveText(
+    "Juan Pablo Montoya"
+  );
+  expect(formRunner.summaryAnswer(formData.emailAddress.question)).toHaveText(
+    "testing@testing.com"
```

Comment:
On the absolute minor offchance we ever do anything with these emails like accidentally send out email notifications, it's safer to use `example.com`, `example.net` and `example.org` for test data - they're reserved domains that can be used for test data, whereas testing.com is a real address 😄

Reply:
@gstevenson fixed